src/test/java/uk/gov/hmcts/reform/jobscheduler/SampleData.java

```diff
@@ -53,4 +53,21 @@ public static Job validJob() {
             )
         );
     }
+
+    public static Job validJobEmptyInterval() {
```

Comment:
Do we need this method? Maybe we could use `validJob()` and set its interval and frequency to null? We're going to remove those fields, as they're not used. There will be just one case.

Reply:
once #44 is merged - `validJob` can be used. cannot set final properties

Reply:
It is merged now